src/pudl/metadata/resources/eia923.py

```diff
@@ -958,6 +958,51 @@
         "sources": ["eia923"],
         "etl_group": "eia923",
     },
+    "_core_eia923__emissions_control": {
+        "description": {
+            "additional_summary_text": (
+                "Actual rate and removal efficiency for air emissions reported by "
+                "thermoelectric or combined-cycle power plants with a total steam turbine capacity "
+                "greater than or equal to 10 megawatts."
+            ),
+            "additional_source_text": "(Schedule 8C)",
+        },
+        "schema": {
+            "fields": [
+                "plant_id_eia",
+                "report_year",
+                "acid_gas_removal_efficiency",
+                "annual_nox_emission_rate_lb_per_mmbtu",
+                "data_maturity",
+                "environmental_equipment_name",
+                "fgd_electricity_consumption_mwh",
+                "fgd_sorbent_consumption_1000_tons",
+                "hours_in_service",
+                "mercury_control_id_eia",
+                "mercury_emission_rate_lb_per_trillion_btu",
+                "mercury_removal_efficiency",
+                "nox_control_id_eia",
+                "operational_status",
+                "ozone_season_nox_emission_rate_lb_per_mmbtu",
+                "particulate_control_id_eia",
+                "particulate_emission_rate_lb_per_mmbtu",
+                "particulate_removal_efficiency_tested",
+                "particulate_removal_efficiency_annual",
+                "particulate_test_date",
+                "so2_control_id_eia",
+                "so2_removal_efficiency_tested",
+                "so2_removal_efficiency_annual",
+                "so2_test_date",
+            ],
+            "primary_key": [
+                "plant_id_eia",
+                "report_year",
+            ],
```

Comment:
At this point in the table's processing, I don't think we should expect it to have a natural primary key. My guess is that if we normalized it into several narrower tables, each of which pertained to just a single type of emissions control equipment with its own ID, each of them would have a natural primary key of `(report_date, plant_id_eia, equipment_id)` but lots of plants won't have lots of kinds of equipment, so looking at all those IDs together in the wide unnormalized table will include a bunch of NA values.